docs/eventing/samples/kafka/channel/000-ksvc.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: serving.knative.dev/v1alpha1
```

Comment:
It looks like most of the other samples use `serving.knative.dev/v1`: https://github.com/knative/docs/search?q=kind%3A+Service&unscoped_q=kind%3A+Service

We should probably be consistent here unless we know it's broken.